web/docs/understand/formats/zeek.md

```diff
@@ -62,8 +62,15 @@ You can import this log as follows:
 vast import zeek < conn.log
 ```
 
+:::info type mapping
+The `import zeek` command implicitly
+[aliases](/docs/understand/data-model/type-system.md) the types `count`, `real`,
+and `addr` from the Zeek TSV `#types` header line to VAST's basic types
+`uint64`, `double`, and `ip`.
```

Comment:
Note to the reviewers: This is actually not the case right now, but I think it should be.
What happens instead is that the types are just rewritten to the corresponding basic types instead of aliases.

Reply:
Are you saying that the Zeek reader should create `count`, `real`, and `addr` aliases? I think that's actually a good idea. This way, Zeek users could writes queries with `:addr`, which is both idiomatic and implicitly restricted to Zeek data.

Reply:
This is impossible as long as we're still supporting the old type names for backwards compatibility.

This comment in the docs should be removed for now, I think.

Reply:
I still think it would be good to be explicit about the relationship between Zeek and VAST types to avoid uncertainty for the reader.

Reply:
I think we should still change the zeek reader after the deprecation period, but I applied the change for now.

Reply:
I agree! But right now it's too much of a conflict, and I wouldn't want to bring out another Threat Bus release just for this.